docs/HowTo/Find-and-Connect/Managing-Peers.md

```diff
@@ -1,49 +1,54 @@
+## No discovery
 
-## Monitoring Peer Connections
+To disable P2P peer discovery, use the
+[`--discovery-enabled`](../../Reference/CLI/CLI-Syntax.md#discovery-enabled) option.
 
-JSON-RPC API methods to monitor peer connections include: 
+With discovery disabled, peers that have already discovered or are otherwise configured to connect
+to the local node (for example, using
+[`admin_addPeer`](../../Reference/API-Methods.md#admin_addpeer)) can initiate connections.
+[Static nodes](Static-Nodes.md) can also initiate connections.
+
+## Monitoring peer connections
+
+JSON-RPC API methods to monitor peer connections include:
 
 * [`net_peerCount`](../../Reference/API-Methods.md#net_peercount)
 * [`admin_peers`](../../Reference/API-Methods.md#admin_peers)
-* [`debug_metrics`](../../Reference/API-Methods.md#debug_metrics)
+* [`debug_metrics`](../../Reference/API-Methods.md#debug_metrics).
+
+## Node connections
 
-## Node Connections
+The default logging configuration does not list node connection and disconnection messages.
 
-The default logging configuration does not list node connection and disconnection messages.  
+To enable listing of node connection and disconnection messages, specify the
+[`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) option `--logging=DEBUG`. For more
+verbosity, specify `--logging=TRACE`.
 
-To enable listing of node connection and disconnection messages, specify the 
-[`--logging`](../../Reference/CLI/CLI-Syntax.md#logging) command line option `--logging=DEBUG`.
-For more verbosity, specify `--logging=TRACE`.  
+The console logs connection and disconnection events when the log level is `DEBUG` or higher. If
+the message `Successfully accepted connection from ...` displays, connections are getting through
+the firewalls.
 
-The console logs connection and disconnection events when the log level is `DEBUG` or higher.  
-If `Successfully accepted connection from ...` is displayed, connections are getting through the firewalls. 
+!!! example "Sample log output"
 
-!!! example "Example log output"
-    `2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO  | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c`
+    `2018-10-16 12:37:35.479-04:00 | nioEventLoopGroup-3-1 | INFO | NettyP2PNetwork | Successfully accepted connection from 0xa979fb575495b8d6db44f750317d0f4622bf4c2aa3365d6af7c284339968eef29b69ad0dce72a4d8db5ebb4968de0e3bec910127f134779fbcb0cb6d3331163c`
 
-## Limiting Remote Connections 
+## Limiting remote connections
 
-In private networks with a level of trust between peers, enabling the [remote connection limits](../../Reference/CLI/CLI-Syntax.md#remote-connections-limit-enabled)
-is unnecessary and disabling may increase the speed at which nodes can join the network.
+In private networks with a level of trust between peers, enabling the
```

Comment:
no fix to do here in this PR, but can we create a ticket to say that maybe we should add the info about why limiting remote connections and not only when not limiting?

Reply:
Done.